tests/helpers/test_location.py

```diff
@@ -43,3 +43,64 @@ def test_closest_returns_closest():
     state2 = State("light.test", "on", {ATTR_LATITUDE: 125.45, ATTR_LONGITUDE: 125.45})
 
     assert state == location.closest(123.45, 123.45, [state, state2])
+
+
+async def test_coordinates_function_as_attributes(hass):
+    """Test coordinates function."""
+    hass.states.async_set(
+        "test.object", "happy", {"latitude": 32.87336, "longitude": -117.22943}
+    )
+    assert location.find_coordinates(hass, "test.object") == "32.87336,-117.22943"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5801702)